utils/common/src/main/kotlin/PluginManager.kt

```diff
@@ -46,6 +46,12 @@ interface Plugin {
      * The type of the plugin.
      */
     val type: String
+
+    /**
+     * A flag to indicate whether the plugin should be enabled by default or not.
+     */
+    val isEnabledByDefault: Boolean
```

Comment:
This property needs a mechanism for enabling a plugin. IIUC such mechanism is implemented only for  package mangager plugins. Would it thus make sense to move this property to `PackageManagerFactory`?

Reply:
> This property needs a mechanism for enabling a plugin.

Why? They're enabled by default.

> Would it thus make sense to move this property to `PackageManagerFactory`?

No, this should be a generic aspect of any plugin. In the future, I see more plugin types making use of this feature.